index.js

```diff
@@ -99,6 +99,17 @@ module.exports = function(opts = {}) {
       };
     },
 
+    resolveFileUrl(x){
+      // "fileName: The path and file name of the emitted asset, relative to output.dir without a leading ./."
+
+      // In most cases it is also relative to html file where script is executes as html stored in root directory
+      /* Also we do not need to convert relative URL to absolute using 'new URL(relative_url,document.baseURI).href'
+         because worker constructor are fine with relative one */
+      
+      // FIXME: This is probably not generic, lol.
+      return `"./${x.fileName}"`;
```

Comment:
One thing to note here: If a user places entry files and chunk files into different folders by using something like `chunkFileNames: 'chunks/[name|.js`, then this will not resolve imports of entry points from chunks correctly. This is not really a problem, though, because at the moment, the plugin would not work anyway in such a scenario: The loader code itself cannot resolve relative imports of chunks from different folders correctly.